utils/batchmanager.py

```diff
@@ -44,8 +44,15 @@ def DefineOptions(self):
                                 dest="negate", default=False,
                                 help="create jobs, but does not submit the jobs.")
         self.parser_.add_option("-b", "--batch", dest="batch",
-                                help="batch command. default is: 'bsub -q 8nh < batchScript.sh'. You can also use 'nohup < ./batchScript.sh &' to run locally.",
-                                default="bsub -q 8nh < ./batchScript.sh")      
+                                help="""batch command to submit job. 
+                                    ==> OLD!!!! LSF submission to a queue, e.g. 8nh:
```

Comment:
why keeping LSF stuff? it seems to me that the supporting shell script has been removed.
Was LSF fully decommissioned ? If yes, all LSF stuff should just go